src/bluehawk/bluehawk.ts

```diff
@@ -24,6 +26,7 @@ interface BluehawkConfiguration {
 }
 
 type ParseAndProcessOptions = ProcessOptions & {
+  reporter?: ActionReporter;
```

Comment:
I think this is the only place where `reporter` is optional - was that intentional?